Gemfile

```diff
@@ -37,22 +37,28 @@ gem 'jbuilder', '~> 2.5'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
-gem 'listen', '>= 3.0.5', '< 3.2'
+# Needs platform to avoid x86_64 in Gemfile.lock
+#gem 'nokogiri', platforms: :ruby
+gem 'nokogiri', platforms: [:mri, :mingw, :x64_mingw]
+# This is needed for other arches to build nokogiri from source
+gem 'mini_portile2'
```

Comment:
I think if you specify the nokogiri platform correctly, you'll get mini_portile2 automatically.